lighty-modules/lighty-netconf-sb/pom.xml

```diff
@@ -83,18 +79,38 @@
             <artifactId>singlenode-configuration</artifactId>
             <scope>test</scope>
         </dependency>
-      <dependency>
-        <groupId>org.opendaylight.mdsal.binding.model.ietf</groupId>
-        <artifactId>rfc8342-ietf-origin</artifactId>
-          <version>10.0.0</version>
-        <scope>test</scope>
-      </dependency>
         <dependency>
-            <groupId>org.opendaylight.netconf</groupId>
-            <artifactId>ietf-netconf-with-defaults</artifactId>
-            <version>4.0.0-SNAPSHOT</version>
+            <groupId>org.opendaylight.mdsal.binding.model.ietf</groupId>
+            <artifactId>rfc8342-ietf-origin</artifactId>
+            <version>10.0.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.netconf.model</groupId>
+            <artifactId>rfc6243</artifactId>
+            <version>4.0.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.opendaylight.netconf.model</groupId>
+            <artifactId>rfc6241</artifactId>
+            <version>4.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.netconf.model</groupId>
+            <artifactId>rfc6470</artifactId>
+            <version>4.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.netconf.model</groupId>
+            <artifactId>rfc6022</artifactId>
+            <version>4.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.netconf.model</groupId>
+            <artifactId>rfc8526</artifactId>
+            <version>4.0.0</version>
+        </dependency>
```

Comment:
Remove rfc8526 dependency. It is duplicated with line 58.

Reply:
Removed